README.md

```diff
@@ -7,10 +7,10 @@ Cruise Control for Apache Kafka
 
 ### Introduction ###
   Cruise Control is a product that helps run Apache Kafka clusters at large scale. Due to the popularity of 
-  Apache Kafka, many companies have bigger and bigger Kafka clusters. At LinkedIn, we have ~7K+ Kafka brokers, 
+  Apache Kafka, many companies have increasingly large Kafka clusters with handrunds of brokers. At LinkedIn, we have ~10K+ Kafka brokers, 
```

Comment:
"hundreds"
And while you're at it, please remove the tilde since it's redundant with the "+" (i.e., either "~10k" which means "around 10k" or "10k+" which means "more than 10k").